src/BenchmarkDotNet/Encodings/EncodingInfo.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Text;
+
+namespace BenchmarkDotNet.Encodings
+{
+    public class EncodingInfo
+    {
+        public static Encoding CurrentEncoding { get; set; } = Encoding.ASCII;
```

Comment:
If I understand correctly, you introduced this static property because now we have this line in the template:
```
BenchmarkDotNet.Encodings.EncodingInfo.CurrentEncoding = $Encoding$;
```
Maybe we can find a better way to pass encoding to a generated project. @adamsitnik what do you think?

Reply:
MultiEncodingString is also using this property.

Reply:
I think there is another one way to pass encoding to a generated project - by using EngineParameter. But even in that case we use CodeGenerator to provide encoding to EngineParameters ctor in BenchmarkType template.